lib/apytypes/_apytypes.pyi

```diff
@@ -906,6 +906,24 @@ class APyFloat:
         """
 
     def to_bits(self) -> int: ...
+    def next_up(self) -> APyFloat:
+        """
+        Get the smallest floating-point number in the same format that compares greater.
+
+        Returns
+        -------
+        :class:`APyFloat`
+        """
+
+    def next_down(self) -> APyFloat:
+        """
+        Get the smallest floating-point number in the same format that compares less.
```

Comment:
```suggestion
        Get the largest floating-point number in the same format that compares less.
```
?

Reply:
It can be a bit confusing talking small and big numbers when negative numbers are involved, maybe one should be more explicit and say: "*Get the floating-point number with the smallest magnitude, in the same format, that compares less.*"?

Reply:
But that number will be zero...

Reply:
Ugh sorry, I meant *largest* instead of *smallest* here. Still think it's good to mention magnitude, so "*"Get the floating-point number with the **largest** magnitude, in the same format, that compares less.*".